<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>cv</title>
  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <style type="text/css">
  /*
   * Copyright 2013 Christophe-Marie Duquesne <chmd@chmd.fr>
   *
   * CSS for making a resume with pandoc. Inspired by moderncv.
   *
   * This CSS document is delivered to you under the CC BY-SA 3.0 License.
   * https://creativecommons.org/licenses/by-sa/3.0/deed.en_US
   */
  
  /* Whole document */
  body {
      font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
      max-width: 800px;
      margin: auto;
      background: #FFFFFF;
      padding: 10px 10px 10px 10px;
  }
  
  /* Title of the resume */
  h1 {
      font-size: 55px;
      color: #757575;
      text-align:center;
      margin-bottom:15px;
  }
  /* h1:hover { */
  /*     background-color: #757575; */
  /*     color: #FFFFFF; */
  /*     text-shadow: 1px 1px 1px #333; */
  /* } */
  
  /* Titles of categories */
  h2 {
      /* This is called "sectioncolor" in the ConTeXt stylesheet. */
      color: #397249;
  }
  /* There is a bar just before each category */
  h2:before {
      content: "";
      display: inline-block;
      margin-right:1%;
      width: 16%;
      height: 10px;
      /* This is called "rulecolor" in the ConTeXt stylesheet. */
      background-color: #9CB770;
  }
  /* h2:hover { */
  /*     background-color: #397249; */
  /*     color: #FFFFFF; */
  /*     text-shadow: 1px 1px 1px #333; */
  /* } */
  
  /* Definitions */
  dt {
      float: left;
      clear: left;
      width: 17%;
      font-weight: bold;
  }
  dd {
      margin-left: 17%;
      margin-bottom:7px;
  }
  p {
      margin-top:0;
      margin-bottom:7px;
  }
  
  /* Blockquotes */
  blockquote {
      text-align: center
  }
  
  /* Links */
  a {
      text-decoration: none;
      color: #397249;
  }
  a:hover, a:active {
      background-color: #397249;
      color: #FFFFFF;
      text-decoration: none;
      text-shadow: 1px 1px 1px #333;
  }
  
  /* Horizontal separators */
  hr {
      color: #A6A6A6;
  }
  
  table {
      width: 100%;
  }
  </style>
</head>
<body>
<h1 id="burak-ogan-mancarci">Burak Ogan Mancarci</h1>
<table>
<tbody>
<tr class="odd">
<td>Tel: +1 778 995 1591</td>
</tr>
<tr class="even">
<td><a href="mailto:ogan.mancarci@msl.ubc.ca">ogan.mancarci@msl.ubc.ca</a></td>
</tr>
<tr class="odd">
<td><a href="https://oganm.com/">oganm.com</a></td>
</tr>
</tbody>
</table>
<!--Skills
---------
**Programming languages:** R, Python, Matlab.

**Computing Environment:** Linux, Windows.

**Bioinformatics:** Experience with expression and sequencing datasets.

**Laboratory techniques:** Common techniques and cell culture.

**Languages:** Turkish (Mother Tongue), English (Fluent), French (Basic).
-->
<h2 id="education">Education</h2>
<dl>
<dt>2013-2022</dt>
<dd><p><strong>MSc, Bioinformatics</strong>; University of British Columbia (Vancouver, Canada)</p>
<p><em>Thesis title: Identification/validation of cell type marker genes of the brain and their use in estimating cell type proportions in brain samples.</em></p>
</dd>
<dt>2009-2013</dt>
<dd><strong>BSc, Molecular Biology And Genetics;</strong> Bilkent University (Ankara, Turkey)
</dd>
</dl>
<h2 id="work-experience">Work Experience</h2>
<dl>
<dt>2022-Present</dt>
<dd><strong>Bioinformatician</strong> - Pavlidis Lab, University of British Columbia
</dd>
</dl>
<h2 id="research-experience">Research Experience</h2>
<dl>
<dt>2014 - 2022</dt>
<dd><p><strong>MSc Student at UBC Micheal Smith Laboratories</strong> - Supervisor: Dr. Paul Pavlidis</p>
<ul>
<li>Identification/validation of cell type marker genes of the brain and their use in estimating cell type proportions in brain samples (<a href="https://github.com/PavlidisLab/neuroExpressoAnalysis">github.com/PavlidisLab/neuroExpressoAnalysis</a>).</li>
<li>Development of a web application to visualize gene expression in brain cell types (<a href="http://neuroexpresso.org/">neuroexpresso.org</a>).</li>
</ul>
</dd>
<dt>2013</dt>
<dd><p><strong>Rotation at BC Children’s Hospital</strong> - Supervisor: Dr. Wyeth Wasserman</p>
<ul>
<li>Analysis of CAGE data for detection of microRNA transcription start sites.</li>
</ul>
</dd>
<dt>2013</dt>
<dd><p><strong>Rotation at Simon Fraser University</strong> - Supervisor: Dr. Fiona Brinkman</p>
<ul>
<li>Analysis of antisense transcription in genomic islands.</li>
</ul>
</dd>
<dt>2012</dt>
<dd><p><strong>Summer internship at University of Zurich</strong> - Supervisor: Dr. Barbara Tschirren</p>
<ul>
<li>Selective mating of Japanese quails and computational analysis of various properties of quail and tit eggs.</li>
</ul>
</dd>
<dt>2011</dt>
<dd><p><strong>Summer internship at Harvard Medical School</strong> - Supervisor: Dr. George Daley</p>
<ul>
<li>Reprogramming of murine and human cells via viral vectors.</li>
</ul>
</dd>
</dl>
<h2 id="teaching-experience">Teaching Experience</h2>
<dl>
<dt>2018</dt>
<dd><strong>Teaching Assistant for Statistical Methods for High Dimensional Biology course (STAT 540) at UBC</strong> - Instructors: Sara Mostafavi, Paul Pavlidis
</dd>
<dt>2016</dt>
<dd><strong>Teaching Assitant for Exploratory Data Analysis course (STAT 545) at UBC</strong> - Instructor: Dr. Jenny Brian
</dd>
<dt>2015</dt>
<dd><strong>Teaching Assitant for Exploratory Data Analysis course (STAT 545) at UBC</strong> - Instructor: Dr. Jenny Brian
</dd>
<dt>2015</dt>
<dd><strong>Instructional Skills Workshop at UBC</strong> - <a href="http://iswnetwork.ca">iswnetwork.ca</a>
</dd>
</dl>
<!--
Awards and Scholarships
--------------------
2016
: 1st place in HackSeq hackaton - [hackseq.com](http://www.hackseq.com/) 

2015
: 3rd place in SportsHack hackaton - [sportshackweekend.org/ca/2015](http://sportshackweekend.org/ca/2015/)

2013-2014
: Canadian Institutes of Health Research Training Program Scholarship

2009-2013
: Bilkent 50% Scholarship
-->
<h2 id="publications">Publications</h2>
<!--zotero cell format -->
<p>Lim, N., Tesar, S., Belmadani, M., Poirier-Morency, G., <strong>Mancarci, B.O.</strong>, Sicherman, J., Jacobson, M., Leong, J., Tan, P. and Pavlidis, P. (2021). Curation of over 10 000 transcriptomic studies to enable data reuse. Database, 2021.</p>
<p>Zhuang, B., <strong>Mancarci, B.O.</strong>, Toker, L. and Pavlidis, P. (2019). Mega-analysis of gene expression in mouse models of Alzheimer’s Disease. Eneuro, 6(6).</p>
<p>Tripathy, S.J., Toker, L., Bomkamp, C., <strong>Mancarci, B.O.</strong>, Belmadani, M., and Pavlidis, P. (2018). Assessing Transcriptome Quality in Patch-Seq Datasets. Front. Mol. Neurosci. 11.</p>
<p>Toker, L., <strong>Mancarci, B.O.</strong>, Tripathy, S., and Pavlidis, P. (2018). Transcriptomic evidence for alterations in astrocytes and parvalbumin interneurons in bipolar disorder and schizophrenia subjects. Biological Psychiatry.</p>
<p><strong>Mancarci, B.O.</strong>, Toker, L., Tripathy, S.J., Li, B., Rocco, B., Sibille, E., and Pavlidis, P. (2017). Cross-Laboratory Analysis of Brain Cell Type Transcriptomes with Applications to Interpretation of Bulk Tissue Data. ENeuro 4.</p>
<p>Tripathy, S.J., Toker, L., Li, B., Crichlow, C.-L., Tebaykin, D., <strong>Mancarci, B.O.</strong>, and Pavlidis, P. (2017). Transcriptomic correlates of neuron electrophysiological diversity. PLOS Computational Biology 13, e1005814.</p>
<p>Horvath, G.A., Demos, M., Shyr, C., Matthews, A., Zhang, L., Race, S., Stockler-Ipsiroglu, S., Van Allen, M.I., <strong>Mancarci, O.</strong>, Toker, L., et al. (2016). Secondary neurotransmitter deficiencies in epilepsy caused by voltage-gated sodium channelopathies: A potential treatment target? Mol. Genet. Metab. 117, 42–48.</p>
<p>Onder, T.T., Kara, N., Cherry, A., Sinha, A.U., Zhu, N., Bernt, K.M., Cahan, P., <strong>Mancarci, B.O.</strong>, Unternaehrer, J., Gupta, P.B., et al. (2012). Chromatin-modifying enzymes as modulators of reprogramming. Nature 483, 598–602.</p>
<h2 id="software">Software</h2>
<p><strong>NeuroExpresso:</strong> A web application for visualization of gene expression data in brain cell types. Available at <a href="http://neuroexpresso.org/">neuroexpresso.org</a>.</p>
<p><strong>markerGeneProfile:</strong> An R package for calculation of marker gene profiles as described in <a href="http://www.eneuro.org/content/4/6/ENEURO.0212-17.2017">Mancarci et al. (2017)</a>. Available at <a href="https://github.com/PavlidisLab/markerGeneProfile">github.com/PavlidisLab/markerGeneProfile</a>.</p>
<p><strong>VASCO:</strong> A web application for visualization of gene expression data from single cell RNA sequencing experiments. Developed for HackSeq 2016. Available at <a href="https://hackseq.github.io/vasco/">hackseq.github.io/vasco</a>.</p>
<p><strong>Viral Voyager:</strong> A web application for visualization and analysis of metavirome data collected for the Tara Oceans Project. Developed for HackSeq 2018. Available at <a href="http://oganm.com/shiny/taracyc/">oganm.com/shiny/taracyc</a>.</p>
<p><strong>Impact Replays:</strong> A web application for visualization play-by-play data from football games. Developed for SportsHack hackaton. Available at <a href="http://daattali.com/shiny/cfl/">daattali.com/shiny/cfl</a>.</p>
<p><strong>eggstractor:</strong> Software to automate extraction of shape and pigmentation related information from quail eggs. Developed for 2012 summer internship under Barbara Tschirren. Available at <a href="https://github.com/oganm/eggstractor">github.com/oganm/eggstractor</a>.</p>
<p><strong>Misc. Bioinformatics related R packages:</strong> I maintain R packages aimed at making life easier for myslef and other bioinformaticians. See <a href="https://github.com/topics/ogan-bio">github.com/topics/ogan-bio</a> for a full list.</p>
<p><strong>Misc. D&amp;D related repositories:</strong> I maintain R packages and web applications to be used while playing Dungeon’s and Dragons, a popular tabletop roleplaying game. See <a href="https://github.com/topics/ogan-dnd">github.com/topics/ogan-dnd</a> for a full list.</p>
<!--
Presentations
--------------------
**Society for Neuroscience Annual Meeting 2017:** **Mancarci, B.O.**, Toker, L., Li, B., Rocco, B., Tripathy, S., Sibille, E., and Pavlidis, P. NeuroExpresso: A brain cell type specific gene expression database composed of pooled microarray and single cell RNA sequencing data.


**25th Annual International Conference on Intelligent Systems for Molecular Biology:** **Mancarci, B.O.**, Toker, L., Li, B., Rocco, B., Tripathy, S., Sibille, E., and Pavlidis, P. (2017). NeuroExpresso: Cross laboratory database of brain cell type specific gene expression.

**Society for Neuroscience Annual Meeting 2016:** Toker, L., **Mancarci, B.O.**, Tripathy, S., and Pavlidis, P. (2016). Deciphering the cell-type specific component in the pathophysiology of brain-related disorders.

**Society for Neuroscience Annual Meeting 2016:** Tripathy, S., Tebaykin, D., **Mancarci, O.**, Toker, L., and Pavlidis, P. (2016). Transcriptomic correlates of brain-wide electrophysiological diversity.

**Society for Neuroscience Annual Meeting 2016:** **Mancarci, O.**, Toker, L., and Pavlidis, P. (2016). Comparison of single cell and pooled cell expression data from mouse and human brain.

**24th Annual International Conference on Intelligent Systems for Molecular Biology:** **Mancarci, O.**, Toker, L., Li, B., Tripathy, S., and Pavlidis, P. (2016). Identification of novel markers for mammalian brain cell types.

**Organization of Computational Neurosciences Conference 2015:** Tripathy, S.J., Tebaykin, D., Li, B., **Mancarci, O.**, Toker, L., and Pavlidis, P. (2015). Large-scale analysis of brain-wide electrophysiological diversity reveals novel characterization of mammalian neuron types. BMC Neurosci 16, O4.

**23rd Annual International Conference on Intelligent Systems for Molecular Biology:** **Mancarcı, O.**, Toker, L., Tripathy, S., Pavlidis, P., Mancarcı, O., Toker, L., Tripathy, S., and Pavlidis, P. (2015). A comprehensive database of cell-type specific&nbsp;marker genes for the mammalian brain. F1000Research 4.

**23rd Annual International Conference on Intelligent Systems for Molecular Biology:** Toker, L., **Mancarci, O.**, Tripathy, S., and Pavlidis, P. (2015). A transcriptomics approach for revealing cell-type proportion changes in psychiatric disorders.
-->
</body>
</html>
